workflow/tools/add_comment.py

```diff
@@ -20,49 +21,48 @@
     'bams': []
 }
 
+with open(task_dict['input'].get('metadata_json'), 'r') as f:
+    metadata = json.load(f)
+
 # the inputs are BAM
 if input_format == 'BAM':
     picard = task_dict['input'].get('picard_jar')
     unaligned_rg_replace_dir = task_dict['input'].get('unaligned_rg_replace_dir')
 
-    with open(task_dict['input'].get('metadata_json'), 'r') as f:
-        metadata = json.load(f)
+    rg_args = []
+    for ct in ['dccProjectCode', 'submitterDonorId', 'submitterSpecimenId', 'submitterSampleId', 'dccSpecimenType', 'libraryStrategy', 'useCntl']:
+        rg_args.append('C=%s:%s' % (ct, metadata.get(ct)))
```

Comment:
where is the code converting camel case to snake case? like: `dccProjectCod` => `dcc_project_code`